src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -65,7 +65,7 @@ public virtual void UpdateAsync(float delta)
         var membrane = organelle!.ParentMicrobe!.Membrane;
 
         // Skip updating if membrane is not ready yet for us to read it and do it in the sync update instead
-        if (membrane.Dirty)
+        if (membrane?.Dirty == true)
```

Comment:
Is there a reason why the membrane can not exist now?

Reply:
Due to how `IGraphicalEntity.RenderPriority` is implemented in Microbe, it tries to access membrane early in its setter before `_Ready` is run.

Reply:
I think a simpler solution would be to for the Microbe class to handle that, wouldn't it be? Similarly to how we have a bunch of controls that allow setting their properties before adding them to the scene and then in their _Ready once they get their sub objects they apply the variables. I suspect that the membrane being nullable requires a ton of code changes, so I think that this might be better to implement as a boolean flag saying if the microbe is initialized yet.

Reply:
Added a check in the setter to not call the applying method if membrane is null.